fedmsg_meta_fedora_infrastructure/tests/releng.py

```diff
@@ -27,6 +27,187 @@
 
 
 class TestTwoWeekAtomicBegin(Base):
+    """ As one of the final steps in the `two week atomic
+    <https://fedoraproject.org/wiki/Changes/Two_Week_Atomic>`_ process, there
+    is a `release engineering script
+    <https://pagure.io/releng/blob/master/f/scripts/push-two-week-atomic.py>`_ that
+    runs to perform the release.
+
+    During the two week script run, it fetches AtomicHost artifacts detail
+    for given pungi compose ID using `fedfind
+    <https://pagure.io/fedora-qa/fedfind/>`_ . After getting image
+    artifacts and ostree commit information successfully for all supported
+    architectures, it sends this fedmsg indicating that Two Week release process has started.
+    """
+
+    expected_title = "releng.atomic.twoweek.begin"
+    expected_subti = "Release engineering scripts started evaluating a " + \
+        "new set of builds from Fedora 22 Atomic Host release"
```

Comment:
22? :)

Reply:
Yeah, looks like it is true. I don't have much information of past on how it got started :)